### PR TITLE
Cleanup documentation

### PR DESCRIPTION
**Pull request by o-rumiantsev**

### CODE REVIEW

File: lib/schema-processor.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ const SCRIPT_TIMEOUT = 2000;
 //   options <Object>
 //     context <Object> object to be assigned to global during loading
 //     decorators <Object> decorators available during schema processing
+//     localDecorators <Object> specific decorators for different
```
Comment (nechaido) [inline lib/schema-processor.js]:
```suggestion
//     localDecorators <Object> type specific decorators
```

File: lib/fs-loader.js [hunk 2]
```diff
@@ -70,60 +70,64 @@ const loadModule = async (dir, options) => {
...
+//     context <Object>
+//     decorators <Object>
 //     localDecorators <Object>, optional
 //       [type] <Object> decorators specific for this schema type
 //     pathToType <Object>
 //       [ext] <string>
 //     pathToType <Function>
 //       path <string>
 //     Returns: <string>
+//     loadOrder <Function>, optional
```
Comment (nechaido) [inline lib/fs-loader.js]: There is no such argument.

File: lib/fs-loader.js [hunk 3]
```diff
@@ -70,60 +70,64 @@ const loadModule = async (dir, options) => {
...
+//       filename1 <string>
+//       filename2 <string>
+//     Returns: <number>
 //     preprocessor: <Function>, optional
-//       schemas <Schema[]>
-//     Returns: <Schema[]>
+//       schemas <Object[]>
```
Comment (nechaido) [inline lib/fs-loader.js]: `<Schema>` is intentional, there is no need to change it to `<Object>` anywhere.

File: lib/fs-loader.js (hunk 2 above)
Comment (o-rumiantsev) [reply lib/fs-loader.js]: https://github.com/metarhia/metaschema/blob/8162a20cb76fea0777926298c15667bedb4a047d/lib/config.js#L130

Comment (nechaido) [reply lib/fs-loader.js]: Remove it, it's not used anywhere.

**Pull request merged**